src/content/operations/nat-traversal/_index.en.md

```diff
@@ -0,0 +1,389 @@
++++
+title = "NAT Traversal"
+date = 2020-08-12T16:02:00+02:00
+weight = 25
++++
+
+## Basic Overview
+
+Submariner establishes the dataplane tunnels between clusters over port
+`4500/UDP` by default. This port can be customized per cluster and per gateway
+and is published as part of the `Endpoint` objects.
+
+### Public vs Private IP
+
+`Endpoint` objects publish both a `PrivateIP` and a `PublicIP`. The `PrivateIP` is the IP assigned
+to an interface on the gateway node where the `Endpoint` originated. The `PublicIP` is the source
+IP for the packets sent from the gateway to the Internet which is discovered by default via
+[ipify.org](https://ipify.org), [my-ip.io](https://my-ip.io) or [seeip.org](https://seeip.org).
+
+Alternative methods can be configured on each gateway via:
+
+```bash
+kubectl annotate node $GW gateway.submariner.io/public-ip=<resolver>,[resolver...]
+```
+
+Resolvers are evaluated one by one, using the result of the first one to succeed.
+`<resolver>` should be written in the following form: `method:parameter`, and the
+following methods are implemented:
+
+| Method   | Parameter                                                 | Notes                                                    |
+|:---------|:----------------------------------------------------------|:---------------------------------------------------------|
+|   api    | HTTPS endpoint to contact, for example api.ipify.org      | The result body is inspected looking for the IP address  |
+|   lb     | LoadBalancer Service name in the Submariner namespace     | A network load balancer should be used                   |
+|   ipv4   | Fixed IPv4 address used as public IP                      |                                                          |
+|   dns    | FQDN DNS entry to be resolved                             | The A entry of the FQDN will be resolved and used        |
+
+For example, when using a fixed public IPv4 address to for a gateway, this can be used:
+
+```bash
+kubectl annotate node $GW gateway.submariner.io/public-ip=ipv4:1.2.3.4
```

Comment:
Just an observation: In the examples below, we are using $NODE and here we are using $GW.

Reply:
Ack I will update all to $GW or $NODE

Reply:
Done